### PR TITLE
feat: add docker command into ai deploy order funnel

### PR DESCRIPTION
**Pull request by abullet33**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #datatr-374 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

customer can add a docker command into ai deploy order funnel for custom app that will be executed on launch 
## Related

Traduction
- [x] CMT-5562

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/ai/apps/add/add.controller.js [hunk 1]
```diff
@@ -323,9 +327,24 @@ export default class AppAddController {
     const lastImagePart = splitImage[splitImage.length - 1];
     const [prefix] = lastImagePart.split(':');
     this.appModel.name = `${prefix}-${nameGenerator()}`;
+    this.appModel.command = this.commandLine
+      ? this.splitIntoArray(this.commandLine)
+      : [];
```
Comment (JacquesLarique) [inline packages/manager/modules/pci/src/projects/project/ai/apps/add/add.controller.js:332]:
```suggestion
    this.appModel.command = this.splitIntoArray(this.commandLine);
```
The test you're making is also done in the splitIntoArray method, so no need to do it again here

**Review by BrunoMarquesOVH — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/ai/apps/add/add.controller.js [hunk 2]
```diff
@@ -323,9 +327,24 @@ export default class AppAddController {
...
+    if (cmd) {
+      return cmd.match(/([^\s"])+|"[^"]+"/g).map((elt) => {
+        if (elt.startsWith('"') && elt.endsWith('"')) {
+          return elt.substring(1, elt.length - 1);
+        }
+        return elt;
+      });
+    }
+    return [];
+  }
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai/apps/add/add.controller.js:346]: **nitpick:** Please use the full-form and avoid shortNames like `cmd`, `elt`

File: packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-image/app-image.controller.js [hunk 3]
```diff
@@ -32,4 +35,22 @@ export default class AppImageController {
...
     return (termsOfService[this.language] || termsOfService.default).url;
   }
+
+  splitStringCommandIntoArray() {
+    if (this.command) {
+      return this.command.match(/([^\s"])+|"[^"]+"/g).map((elt) => {
+        if (elt.startsWith('"') && elt.endsWith('"')) {
+          return elt.substring(1, elt.length - 1);
+        }
+        return elt;
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-image/app-image.controller.js:45]: **nitpick:** Use full-forms and avoid shortNames like `elt`

File: packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-image/app-image.html [hunk 4]
```diff
@@ -205,4 +205,61 @@
...
+                    data-ng-model="$ctrl.command"
+                    data-required
+                />
+            </oui-field>
+        </div>
+        <div class="col-md-4">
+            <oui-tile
+                class="h-600"
+                data-heading="{{:: 'pci_app_docker_command_submit_title' | translate  }}"
+                data-ng-if="$ctrl.customDockerCmd && $ctrl.splitStringCommandIntoArray().length > 0"
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-image/app-image.html:246]:
```suggestion
                data-ng-if="$ctrl.customDockerCommand && $ctrl.splitStringCommandIntoArray().length > 0"
```

File: packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-image/app-image.html [hunk 5]
```diff
@@ -205,4 +205,61 @@
...
+                    data-term="{{:: 'pci_app_docker_command_submit_title_command' | translate  }}"
+                >
+                    <oui-tile-description>
+                        <ul>
+                            <li
+                                ng-repeat="cmd in $ctrl.splitStringCommandIntoArray()"
+                                class="api-list-item"
+                            >
+                                {{ cmd }}
+                            </li>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-image/app-image.html:258]:
```suggestion
                            <li
                                ng-repeat="command in $ctrl.splitStringCommandIntoArray()"
                                class="api-list-item"
                            >
                                {{ command }}
                            </li>
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by jperchoc — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**